mne/decoding/classifier.py

```diff
@@ -453,3 +453,23 @@ def transform(self, epochs_data, y=None):
                                      picks=self.picks, n_jobs=self.n_jobs,
                                      copy=False, verbose=False)
         return epochs_data
+
+
+def compute_patterns(epochs, linear_model):
```

Comment:
so this PR and https://github.com/mne-tools/mne-python/pull/2282 are achieving the same purpose? or are they somehow different?